instrumentation/spring/spring-cloud-gateway/spring-cloud-gateway-webflux-4.3/testing/src/test/java/io/opentelemetry/instrumentation/spring/gateway/v4_3/Gateway43RouteMappingTest.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.gateway.v4_3;
+
+import io.opentelemetry.instrumentation.spring.gateway.common.AbstractRouteMappingTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest(
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+    classes = {Gateway43TestApplication.class})
+class Gateway43RouteMappingTest extends AbstractRouteMappingTest {
+
+  @Test
+  void gatewayRouteMappingTest() {
+    testGatewayRouteMapping();
```

Comment:
any reason you didn't add `@Test` to the super class method?

Reply:
at one point i was overriding the method but thats no longer needed, thanks